src/mjolnir/pbfgraphparser.cc

```diff
@@ -1736,11 +1735,6 @@ struct graph_callback : public OSMPBF::Callback {
           static_cast<Use>(std::stoi(use->second)) == Use::kConstruction) {
         return;
       }
-      // Throw away platforms if include_platforms_ is false
-      if (!include_platforms_ && (use = results.find("use")) != results.end() &&
-          static_cast<Use>(std::stoi(use->second)) == Use::kPlatform) {
-        return;
-      }
```

Comment:
wait.. did you merge master?

Reply:
I clicked auto merge and my code goes messy..